### PR TITLE
Add keebio/iris keymap

### PR DESCRIPTION
**Pull request by nguyenvukhang**
## Description

Adding a keymap for keebio/iris.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/iris/keymaps/khang/keymap.c [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+enum keyboard_layers {
+    _BL = 0,
+    _FN,
+    _SL,
+    _BF,
+    _UP
+};
+
+#include "colors.c"
+#include "extrakeys.c"
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/khang/keymap.c:12]: As a general rule, don't `#include` .c files, only `.h` files.

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/khang/keymap.c (hunk 1 above)
Comment (nguyenvukhang) [reply keyboards/keebio/iris/keymaps/khang/keymap.c]: Sure. Addressed in 6cc7d8d.

File: keyboards/keebio/iris/keymaps/khang/colors.h [hunk 2]
```diff
@@ -0,0 +1,49 @@
+#define MINIMAL_BASEx(hue,sat) \
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/khang/colors.h:1]:
```suggestion
#pragma once

#define MINIMAL_BASEx(hue,sat) \
```

File: keyboards/keebio/iris/keymaps/khang/extrakeys.h [hunk 3]
```diff
@@ -0,0 +1,63 @@
+enum my_keycodes {
```
Comment (fauxpark) [inline keyboards/keebio/iris/keymaps/khang/extrakeys.h:1]:
```suggestion
#pragma once

enum my_keycodes {
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/khang/colors.h [hunk 4]
```diff
@@ -0,0 +1,51 @@
+#pragma once
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/khang/colors.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/keebio/iris/keymaps/khang/config.h [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define EE_HANDS
+#define RGBLIGHT_LAYERS
+#undef RGBLIGHT_ANIMATIONS
+#define RGBLIGHT_MAX_LAYERS 5
+#undef RGBLIGHT_ANIMATIONS
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/khang/config.h:24]:
```suggestion
```

File: keyboards/keebio/iris/keymaps/khang/extrakeys.h [hunk 6]
```diff
@@ -0,0 +1,65 @@
+#pragma once
```
Comment (drashna) [inline keyboards/keebio/iris/keymaps/khang/extrakeys.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/keebio/iris/keymaps/khang/config.h (hunk 5 above)
Comment (nguyenvukhang) [reply keyboards/keebio/iris/keymaps/khang/config.h:22]: resolved in 45f7101

File: keyboards/keebio/iris/keymaps/khang/colors.h (hunk 4 above)
Comment (nguyenvukhang) [reply keyboards/keebio/iris/keymaps/khang/colors.h:17]: resolved in 8963da2

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**